source/index.html.md

```diff
@@ -41,18 +41,28 @@ We expect you to send this information in a field called 'X-Token' inside header
 {
     "home_buyer": {
         "id": "1234",
-        "email": "something@somewhere.co",
+        "email": "home_buyer@email.com",
         "phone": "(123)12341234"
     },
     "events": ["listing_favorited"],
     "delivery_url": "https://api.your_url.com/nestready/events",
     "metadata": "user_id_in_your_system"
 }
+
+//RESPONSE EXAMPLE
+
+{
+    "id": 1,
+    "metadata": "user_1",
+    "delivery_url": "https://api.your_url.com/nestready/events",
+    "events": ['listing_favorited'],
```

Comment:
let's use " instead of '

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/40079286/66431668-6a817980-e9ea-11e9-971e-3e652133bfcf.png)